src/lib/homestore_backend/hs_homeobject.hpp

```diff
@@ -453,12 +453,15 @@ class HSHomeObject : public HomeObjectImpl {
                 REGISTER_COUNTER(snp_dnr_load_bytes, "Loaded bytes in baseline resync");
                 REGISTER_COUNTER(snp_dnr_resend_count, "Mesg resend times in baseline resync");
                 REGISTER_COUNTER(snp_dnr_error_count, "Error times when reading blobs in baseline resync");
-                REGISTER_HISTOGRAM(snp_dnr_blob_process_time, "Time cost(us) of successfully process a blob in baseline resync",
-                   HistogramBucketsType(DefaultBuckets));
+                REGISTER_HISTOGRAM(snp_dnr_blob_process_time,
```

Comment:
I know it is show here for formatting change, can we rename it to _latency, instead of _time?

Reply:
ack